modules/apps_api/db/migrate/20201012135852_remove_permissions_from_directory_applications.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemovePermissionsFromDirectoryApplications < ActiveRecord::Migration[6.0]
```

Comment:
This migration can be removed all together if you change the original migration to create the table without the permissions field

Reply:
Done!